package/yast2-iscsi-lio-server.changes

```diff
@@ -1,4 +1,10 @@
--------------------------------------------------------------------
+--------------------------------------------------------------------
+Wed May 27 15:45:03 CET 2015 - lszhu@suse.de
+
+- remove ":" in target name when identifier is empty (bnc#932170) 
+- 3.1.15
+
+------------------------------------------------------------------
```

Comment:
Just a little thing, please always use the same  number of `dashes`

--------------------------------------------------------------------